### DISCUSSION TITLE
Wondering why my static page isn't updated

### DISCUSSION THREAD

**Discussion by hexico** [vercel/community#1495] [Help] [answered]
### Summary

So I pushed an update to a contact form on my webpage (just changed a single thing in the action) and refreshed an hour later thinking it would update (please correct me if I'm wrong on that) and it didn't.

So I just decided to redeploy the code, I confirmed in the console that it was using the new code I pushed to main.
Still hasn't changed on the static page. I guess I'll refresh in ~4 hours in case it's just an edge cache thing?

Thanks for any help

### Additional information

TLDR;
Website still using old code after redeploying with new code...

**Comment by amyegan** [comment, marked as answer]
Hi @hexico! Do you have any caching rules set for your project? 

The default `Cache-Control` header for static files is `public, max-age=0, must-revalidate` to prevent browsers from caching files locally. This default setting gives users get the latest file from our Global CDN immediately after deploying. If you changed the default with the [Headers](https://vercel.com/docs/concepts/projects/project-configuration#headers) property in your vercel.json file or Middleware, then you might have a local cache problem.

You can find more info about [how caching works for the Edge Network](https://vercel.com/docs/concepts/edge-network/caching#) in the docs and [FAQs](https://vercel.com/docs/concepts/edge-network/frequently-asked-questions#)
